### ISSUE TITLE
No roles found in "My Roles" - Cannot delete failed imports in "My Imports" 

### ISSUE THREAD

**Issue by alessfg** [ansible/galaxy-issues#273]
I am running into two issues with Ansible Galaxy. 

First, I cannot seem to view any of my organizations repositories from within the "My Roles" section in Ansible Galaxy. 

Second, the "My Imports" section seems to have broken. I have two failed imports which I can't seem to get rid of via the web interface (since the roles don't appear on "My Roles") or via the CLI. One of the failed imports corresponds to an import from a repo that never existed but that I was able to import via the CLI whilst conducting some testing. The second failed import corresponds to a repo that I recently deleted and thus no longer exists. 

Any thoughts/ideas on how to fix both of these issues? Is there any way to reset my account to a fresh state?

Thanks in advance!

**Comment by chouseknecht** [comment]
I don't think the failed imports really matter. 

To view repos from your organization, you need to be a collaborator or admin on the repos you wish to import. 

Another potential issue may be the number of repositories you have access to. I don't know what that number is for you, but if it's a large number, then Galaxy won't be able to access and inspect all of the repos. At some point it exceeds the maximum number of API requests, and then nothing ends up displaying. In that case, you have to use the `ansible-galaxy` CLI tool.

**Comment by alessfg** [comment]
Thanks for the quick reply! 

I am a collaborator on both of those repos - I created them! 

I have access to ~65 repos. That surely must be below the limit for API requests? When I first setup my account the repos did appear after all. It was only after I started tweaking with them that they disappeared. 

The `ansible-galaxy` CLI seems to not be working all that well on my end as I can only import organization roles, not delete them. I also need to get rid of both failed imports as they are using Galaxy names that I need to use for other roles.
